types/yarn.lock

```diff
@@ -7,7 +7,7 @@
   resolved "https://registry.yarnpkg.com/@ampproject/remapping/-/remapping-2.2.0.tgz#56c133824780de3174aed5ab6834f3026790154d"
   integrity sha512-qRmjj8nj9qmLTQXXmaR1cck3UXSRMPrbsLJAasZpF+t3riI71BXed5ebIOYwQntykeZuhjsdweEc9BxH5Jc26w==
   dependencies:
-    "@jridgewell/gen-mapping" "^0.1.0"
```

Comment:
Looks like these `types/yarn.lock` changes were accidentally added through find and replace. We should remove these.

Reply:
whoops!